changelog

```diff
@@ -1,3 +1,193 @@
+      * Conditions to word-form numbers in conversations. (@UnorderedSigh)
+      * Disabled fighters and drones will no longer be hit by stray projectiles. (@tibetiroka)
+      * Separate delayed and non-delayed shield generation and hull repair attributes. (@Quantumshark)
+      * Rotate interface images. (@warp-core)
+      * Allow boarding (repair or capture) ships with a fighter flagship. (@tibetiroka)
+      * conditional links. (@Hurleveur)
+      * Game actions can now play music with a `music` node. (@eebop)
+      * Various new cloak related attributes. (@Hurleveur)
+      * Added the "disabled recovery time" attribute which allows a disabled ship to repair itself, with an optional energy and heat cost. (@Azure3141)
+      * Ability to limit turret firing arc. (@1010todd)
+      * Allow plugins to add more metadata to their plugin.txt file. (@Beed-git)
+      * Events can now be made to apply instantly with a delay of 0 days. (@RisingLeaf)
+      * Mission triggers and actions can now give the player debt. (@Amazinite)
+      * Missions can mark systems of importance. (@Amazinite)
+    * Mechanics changes:
+      * Remove the forced-default disabled hails from Government. (@Amazinite)
+      * NPC scan and surveillance personality behavior changes. (@Amazinite)
+      * Auto-pilot now stops thrusting upon reaching 99% of max speed to save energy. (@samrocketman)
+  * User interface:
+    * Include messages when hailing other ships in the message log. (@TomGoodIdea)
+    * The shop now has tooltips for viewing your unabbreviated credits and the names of your ships without needing to select them. (@Amazinite)
+    * Disabled hails are now sent passively. (@bene_dictator)
+    * Hide unknown system names in the escort HUD. (@TomGoodIdea)
+    * Allow multiple pages of controls. (@TomGoodIdea)
+    * A help message for managing multiple ships in the outfitter. (@TomGoodIdea)
+    * Added a preference to turn off the sobel filter on your flagship and target ship in the HUD. (@warp-core)
+    * Added a control for toggling the turret tracking setting. (@TomGoodIdea)
+    * "Fancy" cloaked ship outlines. (@Koranir)
+    * Expand the orbits key image to make space for all vanilla governments. (@lantzk)
+    * Added a camera acceleration setting that causes the camera to shift with the flagship's movement. (@Koranir)
+  * Under the hood:
+    * Improve load panel tooltip timestamp generation. (@RisingLeaf, @ziproot)
+    * Style updates to Armament. (@RisingLeaf)
+    * General purpose TaskQueue instead of SpriteQueue.. (@quyykk)
+    * Fix typo in and add Hungarian language description to Unix metadata file. (@ovari)
+    * Don't recalculate an escort's distance map every frame if its target system already matches the flagship. (@Koranir)
+  * CI/CD and development environment:
+    * Stricter copyright checking. (@tibetiroka)
+    * Don't check for leading/trailing spaces in substitutions defined in missions. (@tibetiroka)
```

Comment:
I think this is duplicated?

Reply:
I'm not sure what you mean. The two lines here are from separate commits: #9750 and #9915.
Do you mean these (or other) entries should be combined?

Reply:
I thought I read this exact line in two spaces, but I can't find it now.